src/plugins/minimal_scene/EngineToQtInterface.cc

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "EngineToQtInterface.hh"
+
+#include <ignition/rendering/Camera.hh>
+#include <ignition/rendering/RenderEngine.hh>
+
+#include <QOpenGLExtraFunctions>
+
+#include <GL/glcorearb.h>
+
+// clang-format off
+namespace ignition
+{
+namespace gui
+{
+namespace plugins
+{
+  /// \brief Private data class for IgnRenderer
+  class EngineToQtInterfacePrivate
+  {
+    /// \brief FBO texture. Stores uploaded GPU -> CPU.
+    /// Used only during fallback.
```

Comment:
What is fallback and when does this happen? Can you add this to the comment?

Reply:
Explained in detail in EngineToQtInterface docs. Thanks.